### PR TITLE
fix: docs/connect/odbc/linux-mac/data-access-tracing-with-the-odbc-driver-on-linux.md

### PR DESCRIPTION
**Pull request by nschonni**
- Add code fence languages
- Trim trailing spaces

### CODE REVIEW

**Comment by PRMerger6** [comment]
@nschonni : Thanks for your contribution! The author, @MightyPen, has been notified to review your proposed change.

File: docs/connect/odbc/linux-mac/data-access-tracing-with-the-odbc-driver-on-linux.md [hunk 1]
```diff
@@ -16,34 +16,35 @@ ms.author: genemi
...
-  
-```  
-$ ODBCSYSINI=/home/myappuser myapp
-```  
-  
+
+Tracing applies to all applications that use the driver in `odbcinst.ini`. To not trace all applications (for example, to avoid disclosing sensitive per-user information), you can trace an individual application instance by providing it the location of a private `odbcinst.ini`, using the `ODBCSYSINI` environment variable. For example:
+
+```bash
+export ODBCSYSINI=/home/myappuser myapp
```
Comment (nschonni) [inline docs/connect/odbc/linux-mac/data-access-tracing-with-the-odbc-driver-on-linux.md]: changed based on the paragraph describing that this is supposed to be set as an environment variable. The value might need to be quoted though

Comment (nschonni) [reply docs/connect/odbc/linux-mac/data-access-tracing-with-the-odbc-driver-on-linux.md]: Actually, this is the inline invokation for the app with the variable set, so i'll undo this

**Comment by PRMerger6** [comment]
@nschonni : Thanks for your contribution! The author, @MightyPen, has been notified to review your proposed change.

**Comment by MightyPen** [comment]
Merging now.

@nschonni Looks fine.  The code block tags are nice to have.  Thanks.

**Pull request merged**